### PR TITLE
Add first pass at sushi-chef-better-world-ed

### PR DESCRIPTION
**Pull request by dkim0526**
##### Description
This PR is to add sushi-chef-better-world-ed by importing content form generated csv of better-world-ed into Kolibri studio.

- Replace souschef with sushi-chef
- Add a thumbnail image
- Update some readme content
- Delete unused components

*Note: There are some content in generated csv file that are invalid - e.g. invalid permission, file not exists. In this case, sushi-chef will skip downloading them.*

### CODE REVIEW

File: sushichef.py [hunk 1]
```diff
@@ -1,33 +1,264 @@
...
+
+    if 'entries' in result:
+        video = result['entries'][0]
+    else:
+        video = result
 
+    video_title = video["title"].capitalize()
+    video_source_id = video_title.strip().replace(" ", "_")
+    video_path = "{}/{}.mp4".format(DOWNLOAD_DIRECTORY, video_id)
+    video_file = files.VideoFile(path=video_path, language=languages.getlang('en').code)
```
Comment (ivanistheone) [inline sushichef.py]: I did a spot check of one video,
http://35.196.143.130/learn/#/topics/c/64a70c91c5fd5f1299b3ef83f0a7503b ,
and I find it is pretty large 100MB+ with resolution 1920x900

We might want to consider downloading a lower-res version from youtube.

Comment (jayoshih) [reply sushichef.py]: Would it be sufficient to just add the compress flag?

File: sushichef.py [hunk 2]
```diff
@@ -1,33 +1,264 @@
...
+        license=CHANNEL_LICENSE
+    )
+    return video_node
+
+def download_document(link):
+    """
+    Downloads pdf from the link
+    and returns document node
+    """
+    if "=HYPERLINK" not in link:
```
Comment (jayoshih) [inline sushichef.py]: Could you add comments on this? It seems like it could be a useful function to reuse in other chefs

**Review by jayoshih — APPROVED** [review]
Nothing super urgent, just one small change

File: sushichef.py (hunk 1 above)
Comment (dkim0526) [reply sushichef.py]: I ran the chef again with `--compress` flag and it seems that videos become smaller now.

File: sushichef.py (hunk 2 above)
Comment (dkim0526) [reply sushichef.py]: Sounds good. I updated this in 36652dd

**Pull request merged**